observation-service/src/main/java/gov/cdc/etldatapipeline/observation/repository/model/dto/ObservationTransformed.java

```diff
@@ -2,10 +2,8 @@
 
 import lombok.*;
 
-@NoArgsConstructor
-@Getter
 @Setter
-@ToString
+@Getter
```

Comment:
If we are using both the @Getter and @Setter then consider replacing them with @Data

Reply:
> If we are using both the @getter and @Setter then consider replacing them with @DaTa

Having `@Data` annotation affects bad for test coverage, so if there is no need to have `hashCode()`, `equals()` and `toString()` it's better to avoid it.